andi/andi.py

```diff
@@ -169,20 +200,17 @@ def build(plan):  # Build all the instances from a plan
     ...     assert b.a is a
     ...     return 'Called with {}, {}, {}'.format(a.value, b.value, non_annotated)
     ...
-    >>> def _get_kwargs(instances, kwarg_types):
-    ...     return {name: instances[cls] for name, cls in kwarg_types.items()}
```

Comment:
Could you please also check other places where a similar code is present? E.g. there is quite a lot of code like this in tests, I think we can replace it. Build functions are another candidates (examples - maybe not all, test utils).

Reply:
Sure, now that the idea sounds good I'm going to work in this kind of things.